galite-core/src/main/kotlin/org/kopi/galite/report/Report.kt

```diff
@@ -229,10 +232,8 @@ abstract class Report : Window() {
   override val model: VReport by lazy {
     initFields()
 
-    genLocalization()
-
     object : VReport() {
-      override var locale: Locale = this@Report.locale ?: ApplicationContext.getDefaultLocale()
+      override val locale: Locale get() = this@Report.locale ?: ApplicationContext.getDefaultLocale()
```

Comment:
override val locale get() = this@Report.locale ?: ApplicationContext.getDefaultLocale()